requirements/edx-sandbox/base.txt

```diff
@@ -8,5 +8,7 @@ numpy==1.6.2
 networkx==1.7
 sympy==0.7.1
 pyparsing==2.0.1
-nltk==2.0.5
 matplotlib==1.3.1
+
+# We forked NLTK just to make it work with setuptools instead of distribute
+-e git+https://github.com/edx/nltk.git@2.0.6#egg=nltk
```

Comment:
Oh, I see this is the sandbox requirements, this should also pin instead of using `-e`